doc/documentation_workflow/developer_documentation_workflow.rst

```diff
@@ -0,0 +1,70 @@
+Developer documentation workflow
+################################
+
+What you need to know
++++++++++++++++++++++
+
+For developer documentation, we use `Doxygen <http://doxygen.org/>`__
+comments extensively throughout NEST.
+
+After installing NEST, you can extract comments from the source code with
+``make doc``. A ``doxygen`` folder with HTML files will be generated in the ``doc`` folder in your source directory.
+
+.. note::
+   This workflow shows you how to create **developer documentation** for NEST. For **user documentation**, please refer to our :doc:`User documentation workflow <user_documentation_workflow>`.
+
+
+Instructions
+++++++++++++
+
+Make sure you have already :doc:`installed NEST <../installation/index>` and created your ``build`` and ``install`` directories. Your ``CMake`` version needs to be up-to-date.
+
+1. Install Doxygen:
+
+.. code-block::
+
+   sudo apt install doxygen
```

Comment:
MacOS: `brew install doxygen` (assuming that brew is already installed)

Reply:
Done.